install/kubernetes/helm/subcharts/operator/templates/clusterrole.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: istio-operator-{{ .Release.Namespace }}
+  labels:
+    app: {{ template "operator.name" . }}
+    chart: {{ template "operator.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+    istio: operator
+rules:
+- apiGroups: [""]
+  resources: ["configmaps"]
+  verbs: ["get", "list", "create", "watch"]
+- apiGroups: ["apiextensions.k8s.io"]
```

Comment:
Will it be easy to give it more persmissions later? What is the permissions upgrade story?

Reply:
@mandarjog Thanks for challenging the assumptions in this PR.  Will verify this works properly.

Reply:
@mandarjog permissions upgrades work for `helm template` + `kubectl apply`.  Will test `helm upgrade` model next.

Cheers
-stedve